docs/variables.md

```diff
@@ -1484,7 +1562,11 @@ but also flushes all L0 files.
 | Data type    | Boolean                            |
 | Default      | OFF                                |
 
-Forces MyRocks to immediately flush all memtables out to data files.
+!!! note
+
+    In version [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md) and later, the default value is changed from `ON` to `OFF`.
+
+This variable acts as a switch. Set the variable to `ON`, `rocksdb_force_flush_memtable_now=ON`, to immediately flush all memtables. After all memtables are flushed, the variable value automatically switches to `OFF`. The default value is `OFF`, which means the variable is disabled.
```

Comment:
This should be better: This variable acts as a trigger. 
Please remove: The default value is `OFF`, which means the variable is disabled.

Reply:
fixed